src/com/facebook/buck/shell/WorkerShellStep.java

```diff
@@ -43,22 +42,16 @@
 public class WorkerShellStep implements Step {
 
   private ProjectFilesystem filesystem;
-  private Path tmpPath;
-  private Path workingDir;
   private Optional<WorkerJobParams> cmdParams;
   private Optional<WorkerJobParams> bashParams;
   private Optional<WorkerJobParams> cmdExeParams;
 
   public WorkerShellStep(
       ProjectFilesystem filesystem,
-      Path tmpPath,
-      Path workingDir,
```

Comment:
You can't remove this, it's not always the root of the filesystem, for `genrule` it's the sandbox directory.